### PR TITLE
[FEATURE] Unique spawning processes

### PR DESCRIPTION
**Pull request by maschwenk**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -193,8 +193,11 @@ export default {
                             .concat(DEFAULT_ARGS, '--stdin', filePath)
         const stdin = editor.getText()
         const cwd = getProjectDirectory(filePath)
-        const { stdout, stderr } = await helpers.exec(command[0], command.slice(1), { cwd, stdin, stream: 'both' })
-        const { files } = parseFromStd(stdout, stderr)
+        const output = await helpers.exec(command[0], command.slice(1), { cwd, stdin, stream: 'both', uniqueKey: 'linter-rubocop' })
```
Comment (Arcanemagus) [inline src/index.js]: Maybe move the options into an `execOptions` object so this line isn't so long?

**Comment by Arcanemagus** [comment]
Note that this will need to depend on the next version of `atom-linter` so it can be sure this is supported.

**Comment by maschwenk** [comment]
@Arcanemagu AFAIK the newest version of Atom Lint supports this.

**Comment by Arcanemagus** [comment]
@maschwenk Correct, I mean that the `package.json` needs to be updated to have `^10.0.0` as the minimum version so you can be sure this is supported.

**Comment by Arcanemagus** [comment]
New minimum was handled in #212, looks good to proceed to me!

**Comment by Arcanemagus** [comment]
@cgalvarez brought up a very good point in https://github.com/AtomLinter/linter-codeclimate/pull/64, as this PR currently implements things you can only be linting one file at a time, the `uniqueKey` needs to be something like `linter-rubocop::${filePath}`.

**Comment by maschwenk** [comment]
Ahhh nice catch

**Comment by Arcanemagus** [comment]
Heh, this is what I get for not jumping on the new functionality myself right away:

Can you think of a reason to keep around the timeout? I'm just disabling the timeout and letting the `uniqueKey` handle it for me.

**Comment by maschwenk** [comment]
yeah I think it's fine to junk it, it was sorta a temp fix until the unique spawning was :shipit:

**Pull request merged**